pages/_app.jsx

```diff
@@ -7,6 +7,9 @@ import {Header} from '../src/components/site/Header';
 import {NavBar} from '../src/components/site/NavBar';
 
 export default ({Component, pageProps}) => {
+	if (Component.noLayout) {
+		return <Component {...pageProps} />;
+	}
```

Comment:
We'll have to find a better solution than that later ! As we said, maybe switch to the `app` folder to use the layouts 🤷🏼‍♂️